src/components/PropEditors.js

```diff
@@ -183,7 +183,7 @@ class NumberEditor extends React.Component {
     return (
       <TextField
         onChange={onChange}
-        value={typeof this.state.value !== "undefined" ? this.state.value : ""}
+        value={typeof this.state.value !== "undefined" && this.state.value !== "null" ? this.state.value : ""}
```

Comment:
That tests for string equality with the string `"null"`.  I did that in the case of the other prop only while the backend is sending such strings.  I think the right comparison here is `!== null`